### PR TITLE
feat(codespaces): update devcontainer.json to pull submodule recursively

### PR DESCRIPTION
**Pull request by Arnabdaz**
Fixes #4009 

#### Describe the changes you have made in this PR -

- [x] add `git submodule update --init --recursive` command for pulling vue simulator

### Screenshots of the changes (If any) -

N/A

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -3,10 +3,10 @@ FROM ruby:3.2.1
 # install dependencies
 RUN apt-get update -qq && apt-get install -y imagemagick shared-mime-info libvips && apt-get clean
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash \
```
Comment (tachyons) [inline .devcontainer/Dockerfile:6]: Could you move this change to another PR ?

Comment (Arnabdaz) [reply .devcontainer/Dockerfile:6]: sure

Comment (Arnabdaz) [reply .devcontainer/Dockerfile]: done

**Comment by Arnabdaz** [comment]
@tachyons this need to be merged else the cv-frontend-vue repo won't be pulled by github codespaces
